itests/kit/ensemble.go

```diff
@@ -136,6 +136,13 @@ type Ensemble struct {
 
 // NewEnsemble instantiates a new blank Ensemble.
 func NewEnsemble(t *testing.T, opts ...EnsembleOpt) *Ensemble {
+	// Ensure we're using the right actors. This really shouldn't be some global thing, but it's the best we can do for now.
+	if build.InsecurePoStValidation {
+		require.NoError(t, build.UseNetworkBundle("testing-fake-proofs"))
+	} else {
+		require.NoError(t, build.UseNetworkBundle("testing"))
+	}
```

Comment:
We used to do this through dependency injection in the test node, but we're setting a global so we might as well do it here.